components/tabs/src/storage.rs

```diff
@@ -32,18 +33,18 @@ pub struct ClientRemoteTabs {
         default = "devicetype_default_deser",
         skip_serializing_if = "devicetype_is_unknown"
     )]
-    pub device_type: DeviceType,
+    pub device_type: TabsDeviceType,
```

Comment:
I think we should try and keep this code using `DeviceType` where we can - the `type` alias above is probably needed to keep uniffi happy, but there should be no need to otherwise touch the .rs.  (IOW, we should see this rename as a short term hack, so should make it as easy to back away from)